edgelet/build/windows/iotedge.wm.xml

```diff
@@ -14,8 +14,8 @@
     <file source="$(_REPO_ROOT)\edgelet\target\$(_Arch)\release\iotedged.pdb" destinationDir="$(runtime.programFiles)\iotedge" />
     <file source="$(_REPO_ROOT)\edgelet\target\$(_Arch)\release\iotedged_eventlog_messages.dll" destinationDir="$(runtime.programFiles)\iotedge" />
     <file source="$(_REPO_ROOT)\edgelet\hsm-sys\azure-iot-hsm-c\build\Release\iothsm.dll" destinationDir="$(runtime.programFiles)\iotedge" />
-    <file source="$(_OPENSSL_ROOT_DIR)\bin\libeay32.dll" destinationDir="$(runtime.programFiles)\iotedge" />
-    <file source="$(_OPENSSL_ROOT_DIR)\bin\ssleay32.dll" destinationDir="$(runtime.programFiles)\iotedge" />
+    <file source="$(_LIBSSL)" destinationDir="$(runtime.programFiles)\iotedge" />
```

Comment:
Since you're switching to the new names, keep the original `$(_OPENSSL_ROOT_DIR)\...` form and just change the filenames.

Reply:
Done.